### ISSUE TITLE
The Moodle Docs page should exist and be kept updated

### ISSUE THREAD

**Issue by cdipe** [wooclap/moodle-mod_wooclap#19]
https://docs.moodle.org/4x/sv/mod/wooclap/mod

**Comment by lucaboesch** [comment]
Probably more like https://docs.moodle.org/en/mod/wooclap @cdipe 
Currently redirected to https://docs.moodle.org/402/en/mod/wooclap
Let's first be the English page there before the Swedish.

**Comment by cdipe** [comment]
@lucaboesch Absolutely, I didn't think of that. A page in English is most important, and probably all that is actually needed.

**Comment by lucaboesch** [comment]
Stub articles added in
https://docs.moodle.org/402/en/Wooclap
and
https://docs.moodle.org/403/en/Wooclap
